media-plugins/vdr-osdteletext/vdr-osdteletext-0.9.7.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user vdr-plugin-2
+
+VERSION="2161" # every bump, new version
+
+DESCRIPTION="VDR Plugin: Osd-Teletext displays the teletext on the OSD"
+HOMEPAGE="https://projects.vdr-developer.org/projects/plg-osdteletext"
+SRC_URI="mirror://vdr-developerorg/${VERSION}/${P}.tgz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+
+DEPEND=">=media-video/vdr-1.7.39"
```

Comment:
This version requirement is always satisfied in ::gentoo and can be lifted off.

Reply:
ok, it's from history. removed version suffix

Reply:
replaced with `DEPEND="media-video/vdr"`